src/main/java/com/autotune/utils/KruizeSupportedTypes.java

```diff
@@ -29,7 +29,7 @@ private KruizeSupportedTypes() { }
 	public static final Set<String> DIRECTIONS_SUPPORTED =
 			new HashSet<>(Arrays.asList("minimize", "maximize"));
 
-	public static final Set<String> MONITORING_AGENTS_SUPPORTED =
+	public static final Set<String> DATASOURCES_SUPPORTED =
```

Comment:
This should be `DATASOURCE_PROVIDERS_SUPPORTED`

Reply:
Updated.